index.html

```diff
@@ -105,7 +105,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-        
+
+        <!-- ________ TheJSPythonator Contributor card START ________  -->
+        <div class="card">
+          <p class="name">TheJSPythonator</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/TheJSPythonator" target="_blank">Github</a>-->
+          </p>
+          <p class="about">Hello! I code in JS and Python. I love reading, writing, drawing and playing Tekken (usually TTT2). I taught myself coding using online resources.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://codetheweb.blog/" target="_blank" title="Code The Web">Code The Web</a>
+              </li>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/" title="MDN Web Docs">MDN Web Docs</a>/a>/a>/a>/a>a>
+              </li>
+              <li>
+                <a href="https://www.sitepoint.com/" target="_blank" title="SitePoint">SitePoint</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________ TheJSPythonator Contributor card END ________  -->
          <!-- ========= Start ========= -->
```

Comment:
move your card after this line where it says, Start
make sure your card has a line break after and before.